src/os/memstore/PageSet.h

```diff
@@ -80,7 +80,14 @@ struct Page {
   Page(char *data, uint64_t offset) : data(data), offset(offset), nrefs(1) {}
 
   static void operator delete(void *p) {
+#if defined(__GNUC__) && !defined(__clang__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
Was this pr https://github.com/ceph/ceph/pull/63796 included? It should have fixed that afair

```
add_library(memstore
  MemStore.cc)
check_cxx_compiler_flag("-Wno-maybe-uninitialized"
```